composer.json

```diff
@@ -21,10 +21,9 @@
         ]
     },
     "require": {
-        "akeneo/pim-community-dev": "dev-master@dev"
+        "akeneo/pim-community-dev": "dev-master"
     },
     "require-dev": {
-        "doctrine/doctrine-migrations-bundle": "1.3.2",
```

Comment:
why ?

Reply:
Becaue it's already a non-dev requirement in CE dev